components/application-operator/charts/application/templates/role-binding.yaml

```diff
@@ -102,6 +102,7 @@ roleRef:
   kind: ClusterRole
   name: {{ .Release.Name }}-application-gateway
 {{- end }}
+{{- if not .Values.global.centralApplicationConnectivityValidatorEnabled }}
```

Comment:
Just a general note. We need to come up with a consistent rollout strategy to ensure smooth migration from Kyma 1.x to 2.0.

Reply:
The value `central_application_connectivity_validator.enabled` set to `true` https://github.com/everesio/kyma/blob/central-application-connectivity-validator/resources/application-connector/values.yaml#L8 will install new chart.

https://github.com/everesio/kyma/blob/central-application-connectivity-validator/resources/application-connector/values.yaml#L11 sets an override for application controller https://github.com/everesio/kyma/blob/central-application-connectivity-validator/resources/application-connector/charts/application-operator/templates/controller.yaml#L58 
This override will automatically deinstall validator resource pro app - migration will be done automatically